src/Compatibility/Microsoft.DotNet.PackageValidation/CompatibleTFMValidator.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.ApiCompatibility;
+using NuGet.ContentModel;
+using NuGet.Frameworks;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.DotNet.PackageValidation
+{
+    /// <summary>
+    /// Validates that there are compile time and runtime assets for all the compatible frameworks.
+    /// Queues the apicompat between the applicable compile and runtime assemblies for these frameworks.
+    /// </summary>
+    public class CompatibleTFMValidator : IPackageValidator
```

Comment:
`CompatibleTfmValidator` ? Usually when an acronym is 3 or more letters, we use PascalCasing.

[https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers)